code/modules/mob/living/carbon/superior_animal/drone/life.dm

```diff
@@ -0,0 +1,7 @@
+/mob/living/carbon/superior_animal/handmade/Life()
+	. = ..()
+
+	if((following) && !(findTarget())) // Are we following someone and not attacking something?
+		while(get_dist(src, following) >= 3) // Keep walking until we get there.
+			step_to(src, following) // Take a step toward the friend
```

Comment:
Remove the while(get_dist(src, following) >= 3) and step_to.

Replace with walk_to(src, following, 4, 5)

The 4 is the distance you have to people (1 = next to them)
The 5 is the speed (experiment with it)

Reply:
Thanks a lot!